README.md

```diff
@@ -64,12 +64,20 @@ This allows you to use llama.cpp compatible models with any OpenAI compatible cl
 
 To install the server package and get started:
 
+Linux
```

Comment:
I believe this also covers MacOS, so should be `Linux / MacOS`

Reply:
yeah wasn't sure. No Mac here